user.go

```diff
@@ -136,12 +138,13 @@ func (c *Client) CurrentUsersTracks() (*SavedTrackPage, error) {
 
 // CurrentUsersTracksOpt is like CurrentUsersTracks, but it accepts additional
 // options for sorting and filtering the results.
+// API Doc: https://developer.spotify.com/documentation/web-api/reference-beta/#endpoint-get-users-saved-tracks
 func (c *Client) CurrentUsersTracksOpt(opt *Options) (*SavedTrackPage, error) {
 	spotifyURL := c.baseURL + "me/tracks"
 	if opt != nil {
 		v := url.Values{}
 		if opt.Country != nil {
-			v.Set("country", *opt.Country)
+			v.Set("market", *opt.Country)
```

Comment:
Good catch - are there other places where we're using `country` and should switch to `market`?

Reply:
On second thought, I'm wondering if we should be reusing the Country option for this.  The country query parameter is more for filtering, while the market parameter does something entirely different.  

If we add support for track relinking, I would want to see all endpoints that support relinking updated.

Reply:
> Good catch - are there other places where we're using `country` and should switch to `market`?

I didn't find any other in _this_ source code.